### PR TITLE
feat: muzzle flash and drawing gun

### PR DESCRIPTION
**Pull request by tehsquidge**

### CODE REVIEW

File: Player.gd [hunk 1]
```diff
@@ -94,7 +94,14 @@ func get_input(delta):
 	elif velocity.x == 0:
 		anim = "idle"
 	
-	sprite.play(anim + "_" + dir)
+	if(
+		anim == "idle" &&
+		sprite.animation == "fire_" + dir &&
+		fire_cooldown_timer > 0
+	):
+		pass
```
Comment (chaegribben) [inline Player.gd:102]: couldn't you just check the opposite?

Comment (tehsquidge) [reply Player.gd:102]: I considered it. But I felt this was more readable at the time. It was late.

Comment (chaegribben) [reply Player.gd:102]: Well it works and we're not being judged on code quality - at least I don't think so - it is a Github hack!
I'll merge it and if you feel like tidying it up on another PR then go ahead :)

**Comment by chaegribben** [comment]
You have conflicts to resolve since your other PR went in

**Pull request merged**